build_release_image.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+set -e
+
+help() {
+
+    cat <<EOM
+Build an Openshift Release Image"
+Usage:                      build_release_image.sh [options] <quay.io username>
+
+Options:
+-h, --help                  show this message
+-t, --tag <tag>             push to a custom tag in your origin release image repo, default: mco
+-m, --mco                   build and push dependancies for mco to quay.io/<username>/image then
+                                build a release image from that repo.
+-c, --capo <ocp certs>      build a capo image locally, push it to quay.io/<username>/capo then
```

Comment:
I strongly suggest keep the same behavior for all image options. It feels weird that --mco just flips a switch to enable the image, while --capo takes an additional argument. It's counter intuitive.

You should probably make the $REPOS_DIR a separate option as it is likely needed to build other images than just CAPO.

Reply:
I think the difference is that its pre-packaged with MCO somehow. You do not need it to build the mco image.